apps/ensindexer/src/lib/plugin-helpers.ts

```diff
@@ -232,3 +232,32 @@ export function networkConfigForContract<CONTRACT_CONFIG extends SubregistryCont
     },
   };
 }
+
+/**
+ * Get the global start block number for each chain ID.
+ *
+ * @returns the global start block number for each chain ID.
+ */
+export async function getIndexingStartBlockNumbersByChainId() {
+  return Object.values((await import("../../ponder.config")).default.contracts)
```

Comment:
Read all contracts configuration returned from `ponder.config.ts` file and then reduce it to get the lowest block height for each network.